bittensor/axon.py

```diff
@@ -405,6 +458,24 @@ def info(self) -> "bittensor.AxonInfo":
             placeholder2=0,
         )
 
+    # Our instantiator should be able to test axon.exception to see if any
```

Comment:
It's better to keep it in docstring. Then it will help not only the developer who reads the code, but also the one who calls `help(function/method/property)`